client/client.go

```diff
@@ -120,3 +122,31 @@ func (c *Client) UpdateGenericSSH(ctx context.Context, inp genericssh.UpdateInpu
 func (c *Client) DeleteGenericSSH(ctx context.Context, inp genericssh.DeleteInput) (*genericssh.DeleteOutput, error) {
 	return genericssh.Delete(ctx, c.client, inp)
 }
+
+func (c *Client) ReadUserByUsername(ctx context.Context, inp user.ReadByUsernameInput) (*user.UserDetails, error) {
```

Comment:
to answer your reply --- I agree that `UserDetails` is indeed a more suitable name as it reflects the content of the return type better, but it is not consistent with other method signature. What about we have it as `xxxOutput` here and assign it as a type alias in the `models.go`? like `type xxxOutput = user.UserDetails` ? It would be immediately clear to the user if they click into the type.